### PR TITLE
Update latest ExCSS.

### LINKED ISSUE #596: Use ExCSS_Core as parser instead of embedded ExCSS code
This is a spin-off of #563 (which grew too long and has more than this subject). 
As discussed [there](https://github.com/vvvv/SVG/issues/563#issuecomment-543711186), it makes sense to use [ExCSS_Core](https://github.com/HockeyJustin/ExCSS_Core) instead of the embedded and changed ExCSS.
As far as I can see, this includes:
- [ ] decide if we will really do this (this may be the only point...)
- [ ] make one or more PRs in ExCSS_Core for fixes/changes that have been made in our code
- [ ] make a PR to add multi-framework support to ExCSS_Core
- [ ] after a new nuget release, remove ExCSS, add the package reference and adapt references where needed 
- [ ] adapt licensing part in Readme

### PR DESCRIPTION
**Pull request by H1Gdev**
#### Reference Issue

Fixes #596

#### Any other comments?

ExCSS version 4.1.4

### CODE REVIEW

File: Tests/Svg.UnitTests/CssQueryTest.cs [hunk 1]
```diff
@@ -78,27 +80,27 @@ private void TestSelectorSpecificity(string selector, int specificity)
         [TestCase("a[id=\"a-02\"]", 0x0110)]
         [TestCase("ul#nav li.active a", 0x1130)]
         [TestCase("body.ie7 .col_3 h2 ~ h2", 0x0230)]
-        [TestCase("#footer *:not(nav) li", 0x1020)]
+        [TestCase("#footer *:not(nav) li", 0x1110)]
```
Comment (H1Gdev) [inline Tests/Svg.UnitTests/CssQueryTest.cs:83]: `not()` is `pseudo-class`.

**Review by mrbean-bremen — APPROVED** [review]
This has been long in the working - thank you for the work you have done to make `ExCSS` compatible with `SVG`, and finally get it in here!

**Comment by mrbean-bremen** [comment]
@H1Gdev - Do you think we are ready for a new release, or do you plan to add something else first? Or maybe anything from the open PRs should be finished first? I very much appreciate your contributions here, I'm mostly concerned with other projects at the moment and not working actively on Svg (and this probably won't change in the foreseeable future).

**Comment by H1Gdev** [comment]
I don't have any PR currently in progress.

**Comment by mrbean-bremen** [comment]
Ok, I published a new release.

**Pull request merged**